### PR TITLE
MGDAPI-3225 Add all rhoam dev team as reviewers for CRO

### PR DESCRIPTION
**Pull request by austincunningham**
## Overview
 Add all rhoam dev team as reviewers for CRO

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -10,9 +10,19 @@ approvers:
 
 
 reviewers:
+- adam-cattermole
 - austincunningham
+- Boomatang
+- carlkyrillos
+- KevFan
 - mikenairn
```
[the remark this answers is not in the document]
Comment (austincunningham) [reply OWNERS]: sure

[the remark this answers is not in the document]
Comment (austincunningham) [reply OWNERS]: taught he might have been in here for some sort of prow job

[the remark this answers is not in the document]
Comment (austincunningham) [reply OWNERS]: Actually looking at the blame Mike was added for `add owners file for prow` will check with Mike to see if he still needs to be there

[the remark this answers is not in the document]
Comment (austincunningham) [reply OWNERS]: @mikenairn do you still need to be in the owners file In CRO for prow jobs ?

[the remark this answers is not in the document]
Comment (mikenairn) [reply OWNERS]: Nope, remove away. :+1:

**Comment by austincunningham** [comment]
Merging this no code verification needed

**Pull request merged**